antiplag/migrations/0007_auto_20201213_1339.py

```diff
@@ -6,13 +6,13 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('antiplag', '0006_auto_20201212_2100'),
+        ("antiplag", "0006_auto_20201212_2100"),
```

Comment:
Thinking whether the migrations should not be ignored by the linter as well.

Reply:
Well migrations will work the same after `black` and will be more readable. Why not?

Reply:
Yeah, they do, but it'll show as change everytime you make new migration and run linter. I would computer generator code as is to avoid unnecessary changes in pull requests.

Reply:
Won't it show as a change if you make new migration and push it to git anyways?

Reply:
Yeah, but only once, when you create it. If some developer forgets to run the linter, the changes will be added to unrelevant commit which ran the linter.

Reply:
Same can happen for any other code, does it mean we won't use lint for other code because of this?  Lint check will be part of pipeline anyways. I think including migrations is fine.